.github/workflows/flutter-format.yml

```diff
@@ -0,0 +1,29 @@
+name: Flutter Format
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - 'client/flutter/**'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/**'
+      - 'client/flutter/**'
+
+jobs:
+  test:
+    name: Format on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Flutter
+        run: |
+          git clone https://github.com/flutter/flutter.git --depth 1 -b stable _flutter
+          echo "::add-path::$GITHUB_WORKSPACE/_flutter/bin"
+      - name: Check format
+        run: flutter format . --set-exit-if-changed --fix
```

Comment:
I think it would be good to just merge this with the test script -- https://github.com/WorldHealthOrganization/app/blob/master/.github/workflows/flutter-test.yml

Reply:
@epicfaace It does not really make a difference as long as it is a separate job because GitHub will display different checks for it always. This way it is more readable. 

If you want to make it one job, I do not think it is a good idea. You want to know when tests are failing or the format is failing separately.

Do you think this is really beneficial? Btw, the only code that your suggestion would save is the trigger code because every code *needs* to do the setup separately. And how would you go about the workflow name?

Reply:
@creativecreatorormaybenot this was merged with the test script in #396.

I think the main reason why it would be good to merge them is because format is a pretty quick step that takes less than a minute. Given that, it doesn't seem to be worth it to add the overhead of setting up flutter, creating an entire new job (especially when GitHub Actions _does_ have a limit on available CI resources), and also maintaining another workflow file.

Your point of the workflow name is well taken, though.